EssayAnalyzer.Api/Services/Foundation/EssayAnalyzes/Exception/EssayAnalysisServiceException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Xeptions;
+
+namespace EssayAnalyzer.Api.Services.Foundation.EssayAnalyzes.Exception
+{
+    public class EssayAnalysisServiceException : Xeption
+    {
+        public EssayAnalysisServiceException(Xeption innerException)
+           : base(message: "OpenAi Service error occured, contact support.", innerException)
+        { }
```

Comment:
"OpenAi Service error occured, contact support." change this to "Essay Analysis  Service error occured, contact support."